client/main/backstage.js

```diff
@@ -1068,18 +1082,34 @@ var BackstageModel = Backbone.Model.extend({
         if (path === '')
             path = this._wdData[wdType].defaultPath;
 
+        let retrievingTimeout = null;
+        retrievingTimeout = setTimeout(() => {
+            let wdData = this._wdData[wdType];
+            for (let model of wdData.models)
+                model.set('retrieving', true );
+            retrievingTimeout = null;
+        }, 100);
```

Comment:
i don't think we need a timeout ... and 100ms would probably be too short. not sure if you're thinking of the web version in writing this, but we'll probably need to come up with a more global approach for that. i think let's not worry about the time out for now.

Reply:
The timeout stops the flicker every time you click on a folder.